Formula/acl2.rb

```diff
@@ -4,45 +4,45 @@ class Acl2 < Formula
   url "https://github.com/acl2/acl2/archive/8.3.tar.gz"
   sha256 "45eedddb36b2eff889f0dba2b96fc7a9b1cf23992fcfdf909bc179f116f2c5ea"
   license "BSD-3-Clause"
-  revision 4
+  revision 5
 
   bottle do
     sha256 "47d523299e219e13e26adb3e3cc4d2eb984e9d535ef242bf16ee4c92229b63aa" => :big_sur
     sha256 "8bfb7ce324dfa93e6d8f46acaafbf69be4ed400c101371af97aa6ff7d9503469" => :catalina
     sha256 "547ba61b3a0514bd55ff3a04b19c4e5792ec19eb64f46cbb672bf30ffd871b93" => :mojave
   end
 
-  depends_on "sbcl"
+  depends_on "clozure-cl"
+  depends_on "openssl@1.1"
+  depends_on "z3"
 
   def install
+    suffix =
+      if OS.mac?
+        "dx86cl64"
+      elsif OS.linux?
+        "lx86cl64"
+      end
```

Comment:
@iMichka do we have a better way here?

Reply:
I mean; ask upstream to make their stuff easier with a plain "make" / "make install" step, as I find it weird that we have to go through so many hoops here to get stuff done. This is all prone to future breakage.

But this should do it:
```
suffix = nil
on_macos do
  suffix = "dx86cl64"
end
on_linux do
  suffix = "lx86cl64"
end
```

Reply:
Okay, I update this formula to run acl2 without the OS condition.
Could you review it again?